flake.nix

```diff
@@ -174,6 +174,7 @@
 
         internalApiDocsConfigureFlags = [
           "--enable-internal-api-docs"
+          "--enable-external-api-docs" # todo: not the right place
```

Comment:
@farcaller we have a new doxygen site that's enabled by this.

We might want to duplicate what we've done for the internal docs, although those aren't properly published anywhere yet.